Added info about manuall Redis cache clearing (#594)
ezsystems/developer-documentation: docs/releases/updating_ez_platform.md

    bin/console ezplatform:migrate:legacy_matrix
    ```

    #### Required manual cache clearing if using Redis

>     If you are using Redis as you persistence cache storage and you see a notice message like `In AbstractTagAwareAdapter.php line 71: Notice: Undefined index: value`, please clear Redis instance manually after the upgrade. You can do it in two ways, by using `redis-cli` and executing following command:

Reviewer: Actually they should not wait for the notice, they MUST clear cache on upgrades in the first place.
But having note here saying that you'll get such notice, and non working cache if you don't is good so people may find the info when they forget.

Author: Indeed, the point was to put the notice message in the docs, so it can be searchable I hope.

>     If you are using Redis as you persistence cache storage and you see a notice message like `In AbstractTagAwareAdapter.php line 71: Notice: Undefined index: value`, please clear Redis instance manually after the upgrade. You can do it in two ways, by using `redis-cli` and executing following command:

Reviewer 2: ```suggestion
    If you are using Redis as your persistence cache storage and you see a notice message such as:
    `In AbstractTagAwareAdapter.php line 71: Notice: Undefined index: value`, clear Redis instance manually after the upgrade.
    You can do it in two ways, by using `redis-cli` and executing the following command:
```

    bin/console ezplatform:migrate:legacy_matrix
    ```

    #### Required manual cache clearing if using Redis

>     If you are using Redis as your persistence cache storage you should always clear it manually after an upgrade.

Reviewer: ```suggestion
    If you are using Redis as your persistence cache storage you should always clear it manually during an upgrade.
```

Unsure, but seems more correct or?
